fleet_notifications/state_checker.py

```
@@ -1,110 +1,152 @@
+    def _check_if_order_is_new(self, car_id: int, state: OrderState, admin_phone: str, under_test: bool) -> None:
+        """Adds the order belionging to the state to the list of orders. If the order is the first new one,
+        a notification is sent to the admin phone number. Returns false if the order can't be retrieved."""
+        no_active_order = True
+        for order in self.orders.values():
+            if order.car_id == car_id:
+                no_active_order = False
+                break
+        if state.order_id not in self.orders or state.status == OrderStatus.CANCELED:
+            try:
+                self.orders[state.order_id] = self.order_api.get_order(car_id=car_id, order_id=state.order_id)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)